alarm/models.py

```diff
@@ -14,26 +14,64 @@ class Phone(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     number = PhoneNumberField(blank=True, unique=True, region='UA')
     enabled = models.BooleanField(default=False)
+    message = models.TextField(null=True, blank=True)
+
+    def get_trade_pairs(self):
```

Comment:
Should be a property.

Reply:
property should be named as an attribute, i.e. do not start with verb, i.e. `trade_pairs`.